licensedb/internal/normalize/normalize.go

```diff
@@ -76,10 +76,10 @@ var (
 	urlCleanupRe = regexp.MustCompile("[<(](http(s?)://[^\\s]+)[)>]")
 )
 
-// NormalizeLicenseText makes a license text ready for analysis.
+// LicenseText makes a license text ready for analysis.
 // It follows SPDX guidelines at
 // https://spdx.org/spdx-license-list/matching-guidelines
-func NormalizeLicenseText(text string, strict bool) string {
+func LicenseText(text string, strict bool) string {
```

Comment:
But it really does normalization. The term is common in data science. The synonym is cleanup.

Reply:
sure, but when you call it you'll write `normalize.LicenseText` since `normalize` is the name of the package.